homework-pages/calculatorStartPage.js

```diff
@@ -0,0 +1,24 @@
+exports.CalculatorStartPage = class CalculatorStartPage {
+    constructor(page) {
+        this.page = page;
+    }
+
+    async goto() {
+    await this.page.goto('https://testsheepnz.github.io/BasicCalculator');
+    }
+    async selectBuild(buildType) {
+        await this.page.click('#selectBuild');
```

Comment:
It's a good practice to separate selectors from the code like you did in CalculateResutsPage.js